src/components/jogging/JoggingCartesianAxisControl.tsx

```diff
@@ -39,7 +43,27 @@ export const JoggingCartesianAxisControl = externalizeComponent(
         element.textContent = displayValue
       })
       const theme = useTheme()
-      const [borderColor, setBorderColor] = useState(colors?.borderColor)
+
+      const forceRenderPressed =
+        renderIncrementPressed || renderDecrementPressed
+      console.log(
+        renderIncrementPressed,
+        renderDecrementPressed,
+        forceRenderPressed,
+      )
+      const [borderColor, setBorderColor] = useState(
```

Comment:
rather than setting the border color as a state, it'd be a bit clearer to set the "pressed" state as a boolean and then compute the border color based on this